### DISCUSSION TITLE
React: Adding Components Plugins in React

### DISCUSSION THREAD

**Discussion by GytisKau** [fancyapps/ui#167] [Q&A] [answered]
Hello,

First of all, this project is AMAZING and I love every single line of code here, also the Documentation is just perfect. 

I'm using the [@fancyapps/ui](https://www.npmjs.com/package/@fancyapps/ui) package in my React project and set up everything as in the react documentation [example](https://fancyapps.com/docs/ui/fancybox/react). When using the `<Fancybox>` component it seems that the thumbnails Plugin is deactivated and I'm not educated enough to turn it on myself. So if someone could show me how to fix this, it would be great.

 Thank you so much!

**Comment by fancyapps** [comment, marked as answer]
Hi,

Did you check the Codesandbox demo https://codesandbox.io/s/fancybox-react-demo-9k2kz?file=/src/App.js ? It works fine, I do not know how to help you.

**Reply by GytisKau** [reply]
Ok, turns out replacing image elements with divs and background images was not the best idea 😅. Everything works fine now
Thanks for your help
